### DISCUSSION TITLE
Listing Stats via API?

### DISCUSSION THREAD

**Discussion by bernasb** [etsy/open-api#1150] [Q&A]
Is there a product stats extraction for individual listings?  Can I 'GET' info from a listing page such as this, for example?

Sample formatted URL (not a real product):  https://www.etsy.com/your/shops/me/stats/listings/1234567890?date_range=all_time

Listing pages like this have extensive time-series data that can be summarized and fed back into listing SEO.

**Comment by anastasiam** [comment]
Hello! I'm also intrested in getting stats data through API by date range. Like the one that you can see in Listing Stats page.

**Comment by jvlobo** [comment]
@bernasb did you find out something? There is no information an no-one from Etsy reply here or in the support email :(

**Comment by linda-dge** [comment]
I would love this feature as well.
